acnportal/acnsim/models/evse.py

```diff
@@ -115,18 +125,26 @@ def set_pilot(self, pilot, voltage, period):
             if self._ev is not None:
                 self._ev.charge(pilot, voltage, period)
         else:
-            raise InvalidRateError('Pilot {0} A is not valid for for station {1}'.format(pilot, self.station_id))
+            raise InvalidRateError(f"Pilot {pilot} A is not valid for "
+                                   f"station {self.station_id}.")
 
-    def _valid_rate(self, pilot):
+    def _valid_rate(self, pilot, atol=1e-3):
```

Comment:
You are correct that atol is never used here by the code itself. However, I think for clarity, if we anticipate atol being a keyword argument in all subclasses of BaseEVSE, we should provide it as a keyword argument in the base class with a suggested default value. Alternatively, we can make atol a required input (i.e. remove the `=1e-3`) or, we can put the atol keyword argument only in inherited classes. If we want to use this last method, we need to provide **kwargs to the _valid_rate method signature in BaseEVSE to avoid violating the LSP.

Reply:
@zach401 let me know which method you like best. I'm leaning towards keeping it as it is, but I don't have strong feelings either way.

Reply:
I think as-is is fine.